packages/ai/core/tool/mcp/mcp-client.ts

```diff
@@ -235,12 +268,6 @@ class MCPClient {
     params?: PaginatedRequest['params'];
     options?: RequestOptions;
   } = {}): Promise<ListToolsResult> {
```

Comment:
should there be an assertCapability call here? or is this now handled downstream?

Reply:
it's handled downstream in request method!